src/screens/AddSiteFormScreen.js

```diff
@@ -104,19 +104,13 @@ class AddSiteFormScreen extends Component {
 
     accessibilityOptions() {
         return Object.keys(accessibility_options).map((key) => {
-            return <Picker.Item key={key} label={accessibility_options[key]} value={accessibility_options[key]}/>;
-        })
-    }
-
-    facilitiesOptions() {
-        return Object.keys(facilities_options).map((key) => {
-            return <Picker.Item key={key} label={facilities_options[key]} value={facilities_options[key]}/>;
+            return <Picker.Item key={key} label={accessibility_options[key]} value={key}/>;
         })
     }
 
     priceOptions() {
         return Object.keys(price_options).map((key) => {
-            return <Picker.Item key={key} label={price_options[key]} value={price_options[key]}/>;
+            return <Picker.Item key={key} label={price_options[key]} value={key}/>;
```

Comment:
This is the fix for GH-56: Instead of recording the string version of the text in the `value`, we just put the key. This keeps us future-proof in case we decide to slightly change the text display, we can still match on the keys. @amgoodman5 however it's REALLY important that we don't change any of the text keys or we'll have to go back and update all values in the firebase db. The nice thing, however, is that if we keep the keys the same, we can change the value of the strings at any time and everything will still match.